rpa-manager/modules/ROOT/pages/processautomation-manage.adoc

```diff
@@ -70,18 +71,24 @@ For an explanation of the properties, see <<form-createprocess, *Create process*
 
 To complete copying the project:
 
-. Transfer the project to the *Build* phase.
-. Open it in the xref:rpa-builder::index.adoc[RPA Builder].
+. Transfer the project copy to the *Build* phase.
+. Open the project copy in the xref:rpa-builder::index.adoc[RPA Builder].
+. xref:rpa-builder::ui-overview#menu-bar[Upload] the project copy to RPA Manager.
+
+The project copy then receives its own identification values. 
```

Comment:
```suggestion
The project copy receives its own identification values. 
```

Reply:
Then and only then. This lead to confusion among users in the past. So I will keep it.